examples/tutorial.dx

```diff
@@ -580,13 +580,12 @@ z
 temptab
 
 ' Using this construct we can return to extracting
-  the 5's from the image set. Here `mempty` is
-  synonymous with `AsList _ []`.
+  the 5's from the image set.
 
 def findFives (x : a=>b) (y : a=>Class) : List b =
     concat for i. case (y.i == (5 @ _)) of
-            True -> AsList _ [x.i]
-            False -> mempty
+            True -> toList [x.i]
+            False -> toList []
```

Comment:
Instead of `toList []` (heavyweight) or `(mempty: List a)` (`Monoid`-specific), how about adding `nil` to the Prelude?

`nil` added in https://github.com/google-research/dex-lang/pull/473.

Reply:
Oh, I did this on purpose. I was trying to be verbose but clear what was going on.

Reply:
Actually, no action requested here. https://github.com/google-research/dex-lang/pull/473 (addition of `nil` to Prelude) can handle rewriting to `nil`.

The only `toList [x.i]` call is great!